definitions/SpellMisc.dbd

```diff
@@ -16,9 +16,9 @@ float LaunchDelay
 int<FileData::ID> SpellIconFileDataID
 float MinDuration
 int<ContentTuning::ID> ContentTuningID
-int<PlayerCondition::ID> PlayerConditionID
+int<PlayerCondition::ID> ShowFutureSpellPlayerConditionID
```

Comment:
What *is* a future spell?!

Reply:
Its all the spells displayed in spell book when you are too low level to know them yet.
This condition is used for spells learned from quests in Exile's Reach, each class has one